ols/src/tools/oc_cli.py

```diff
@@ -0,0 +1,238 @@
+"""OpenShift CLI tools."""
+
+import logging
+import os
+import subprocess
+from typing import Optional
+
+from langchain.tools import tool
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: might also sanitize the case when llm sends the full command instead
+# of just args, eg. [oc get pods -n bla] -> [pods -n bla]
+def sanitize_oc_args(args: list[str]) -> list[str]:
+    """Sanitize `oc` CLI arguments."""
+    blocked_chars = {";", "&", "|", "`", "$", "(", ")", "<", ">", "\\"}
+
+    sanitized_args = []
+    for arg in args:
+        safe_part = []
+        for char in arg:
+            if char in blocked_chars:
+                # stop processing further characters in this argument
+                logger.warning(
+                    "Problematic character(s) found in oc tool argument '%s'", arg
```

Comment:
probably we should stop processing such input as is, not try to sanitize it further